### PR TITLE
Am change

### PR DESCRIPTION
**Pull request by ruchiramoitra**
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-transformer/155)

### CODE REVIEW

File: v0/am/transform.js [hunk 1]
```diff
@@ -62,9 +62,15 @@ function fixSessionId(payload) {
     ? stringToHash(payload.session_id)
     : -1;
 }
- function addMinIdlength() {
-   return {"min_id_length":1}
- }
+
+function fixVersion(payload, message) {
```
Comment (sayan-rudder) [inline v0/am/transform.js:66]: can we change only the config jsons and not add here ?
like:
"context.app.build": "app_version", ==> "context.app. version": "app_version"

**Comment by sayan-rudder** [comment]
resolve conflicts

File: v0/am/transform.js (hunk 1 above)
Comment (arnabp92) [reply v0/am/transform.js:66]: The default should be `context.app.build` but for android specifically, it should be `context.app.version`

Comment (sayan-rudder) [reply v0/am/transform.js:66]: 👍

**Comment by sayan-rudder** [comment]
did you run lint ?

**Pull request merged**